.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: 'Build'
         run: |
           mkdir -p wheelhouse && \
-          NVTE_FRAMEWORK=pytorch pip wheel -w wheelhouse . -v
+          NVTE_FRAMEWORK=pytorch MAX_JOBS=1 pip wheel -w wheelhouse . -v
```

Comment:
Why do we need that?

Reply:
It turns out the testing systems that GitHub gives us are wimpy and experience memory issues if we let Ninja go full-force. It seems PyTorch has experienced this kind of problem before:
https://github.com/pytorch/pytorch/blob/c27cefccd371ec451a9be51fc5aced1125791493/torch/utils/cpp_extension.py#L448-L451